Userland/Games/Chess/main.cpp

```diff
@@ -135,24 +142,34 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     engines_action_group.add_action(human_engine_checkbox);
     TRY(engine_submenu->try_add_action(human_engine_checkbox));
 
-    auto action = GUI::Action::create_checkable("ChessEngine", [&](auto& action) {
-        auto new_engine = Engine::construct(action.text());
-        new_engine->on_connection_lost = [&]() {
-            if (!widget->want_engine_move())
-                return;
-
-            auto rc = GUI::MessageBox::show(window, "Connection to the chess engine was lost while waiting for a move. Do you want to try again?"sv, "Chess"sv, GUI::MessageBox::Type::Question, GUI::MessageBox::InputType::YesNo);
-            if (rc == GUI::Dialog::ExecResult::Yes)
-                widget->input_engine_move();
-            else
-                human_engine_checkbox->activate();
-        };
-        widget->set_engine(move(new_engine));
-        widget->input_engine_move();
-    });
-    engines_action_group.add_action(*action);
-
-    TRY(engine_submenu->try_add_action(*action));
+    Vector<EngineDetails, 2> available_engines {
+        { "/bin/ChessEngine"sv, "ChessEngine"sv },
```

Comment:
No need to specify the path here, the executable name is enough.
An uppercase S would be nice.

Reply:
Changed to capital S.